subxt/src/backend/unstable/follow_stream_driver.rs

```diff
@@ -251,7 +250,7 @@ impl<Hash: BlockHash> Shared<Hash> {
                 // New subscriptions will be given this init message:
                 shared.current_init_message = Some(ev.clone());
                 // Clear block cache (since a new finalized block hash is seen):
-                shared.block_events_from_last_finalized.clear();
+                shared.block_events_for_new_subscriptions.clear();
```

Comment:
yeah, alright when `Initialized` event is generated we can conclude that no messages regarding the next block are missed.